src/Http/Http/test/Features/FormFeatureTests.cs

```diff
@@ -83,41 +83,6 @@ public async Task ReadFormAsync_SimpleData_ReturnsParsedFormCollection(bool buff
             await responseFeature.CompleteAsync();
         }
 
-        [Theory]
-        [InlineData(true)]
-        [InlineData(false)]
-        public async Task ReadFormAsync_SimpleData_ReplacePipeReader_ReturnsParsedFormCollection(bool bufferRequest)
```

Comment:
I think we want to keep this test as well. Use the feature instead of the setter.